plugins/node/opentelemetry-instrumentation-restify/README.md

```diff
@@ -0,0 +1,50 @@
+# OpenTelemetry Restify Instrumentation for Node.js
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`restify`](https://github.com/restify/node-restify) and allows the user to automatically collect trace data and export them to their backend of choice.
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-restify
+```
+### Supported Versions
+ - `>=4.0.0`
+
+## Usage
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const RestifyInstrumentation = require('@opentelemetry/instrumentation-restify');
+
+const provider = new NodeTracerProvider();
+provider.register();
+new RestifyInstrumentation();
```

Comment:
trying some unification in examples / readme with `registerInstrumentations` like in core repo (soon in contrib too when plugins will be removed) for example
https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation-http.
Can you please make an example with `registerInstrumentations`

Reply:
Sure! Updated the example.